packages/lib-classifier/src/store/FeedbackStore.js

```diff
@@ -0,0 +1,66 @@
+import { autorun } from 'mobx'
+import { addDisposer, getRoot, types } from 'mobx-state-tree'
+import _ from 'lodash'
+
+import * as helpers from './feedback/helpers'
+import strategies from './feedback/strategies'
+
+const FeedbackStore = types
+  .model('FeedbackStore', {
+    isActive: types.optional(types.boolean, false),
+    rules: types.frozen({})
+  })
+  .actions(self => {
+    function afterAttach () {
+      createSubjectObserver()
+    }
+
```

Comment:
Maybe wire up an `onAction` observer here (as per previous comment) then fire the appropriate feedback actions in response to classifier state changes.

Reply:
I'm inclined to just delete 2d81f7177117d7942ccbe7a22d98365fb19791a7.